### DISCUSSION TITLE
Cannot find the Addon wmbusmeters in Homeassistant add-on-store

### DISCUSSION THREAD

**Discussion by thoath2** [wmbusmeters/wmbusmeters#1375] [General]
Hello,
can anybody help me, where I can find and install the addon "wmbusmeters" in Home Assistant.
I want to use it for read out wired M-Bus-Devices.

I didn't find it in 

- "Einstellungen" / "Add-ons"
- "HACS"

I use:
HAOS  Core 2024.8.2
Supervisor 2024.09.1
Operating System 13.0
Frontend 20240809.0

Thomas

**Comment by BIBOLV** [comment]
Hi,
This addon is not in stores - please visit https://github.com/wmbusmeters/wmbusmeters-ha-addon and install with push of a button or follow manual installation steps.
